src/sass/bulma.scss

```diff
@@ -708,8 +708,9 @@ div.stretchy-wrapper > div.stretchy-text {
 
 // Dark Mode
 html.has-theme-dark {
+  background-color: $black;
   body.has-theme-dark {
-    background-color: $black-bis;
+    background-color: $black;
```

Comment:
Shouldn't the `-bis` stay? Without it, the whole background is just pitch dark. With `-bis`, the footer is offset from the rest of the page. Looks a lot better in my eyes.

Reply:
Yes, you are correct. I accidentally changed both.